crates/lang/codegen/src/generator/events.rs

```diff
@@ -77,6 +77,11 @@ impl<'a> Events<'a> {
     /// Generates the base event enum that comprises all user defined events.
     /// All emitted events are converted into a variant of this enum before being
     /// serialized and emitted to apply their unique event discriminant (ID).
+    ///
+    /// # Developer Note
+    ///
+    /// The `__ink_dylint_EventBase` config attribute is used here to convey the
+    /// information that this function is an ink! constructor to `dylint`.
```

Comment:
Here it shouldn't be constructor, but I'm not sure what you're getting at.

Should it maybe be `generator` instead?